src/socket.c

```diff
@@ -195,6 +195,14 @@ static int raw_connect_internal(struct trilogy_sock *sock, const struct addrinfo
         return TRILOGY_SYSERR;
     }
 
+#ifdef TCP_NODELAY
+    if (sock->addr->ai_family != PF_UNIX) {
```

Comment:
Does this make sense?

Reply:
Looks good to me! We can't and don't need to apply this to unix sockets.